examples/mms/README.md

```diff
@@ -17,6 +22,84 @@ MMS-1B-all| 1162 | MMS-lab + FLEURS <br>+ CV + VP + MLS |  [download](https://dl
 
 \* In the `Dictionary` column, we provide the download link for token dictionary in English language. To download token dictionary for a different language supported by the model, modify the language code in the URL appropriately. For example, to get token dictionary of FL102 model for Hindi language, use [this](https://dl.fbaipublicfiles.com/mms/asr/dict/mms1b_fl102/hin.txt) link. 
 
+**🤗 Transformers**
+
+First, we install transformers and some other libraries
+```
+pip install torch accelerate torchaudio datasets
```

Comment:
As far as I'm aware, we don't need `accelerate` or `torchaudio` no? Note that all audio decoding / resampling is done with librosa/soundfile in datasets now, both of which can be installed with:

```suggestion
pip install torch datasets[audio]
```